src/WebAppDIRAC/WebApp/static/DIRAC/Accounting/classes/Accounting.js

```diff
@@ -91,16 +91,6 @@ Ext.define("DIRAC.Accounting.classes.Accounting", {
             ["JobSplitType", "Job Split Type"],
           ],
         },
-        ComponentMonitoring: {
```

Comment:
This should stay, I think.

Reply:
We won't have a ComponentMonitoring anymore after [#5944](https://github.com/DIRACGrid/DIRAC/pull/5944) gets merged, so unless we replace it with a service- and agentmonitoring in the WebApp I think it needs to be removed.